<?xml version="1.0" ?>
<launch>
    <arg name="app" default="rbpf"/>
	  <arg name="dataset" default="ripples"/>
    <arg name="namespace" default="hugin_0"/>
    <arg name="node_name"/>
    <arg name="gp_plot_server" default="/gp/plot_server"/>
    <arg name="sample_sample_server" default="/gp/sample_server"/>
    
    <node type="particle_map.py" pkg="rbpf_slam" name="$(arg node_name)" output="screen">
      <param name="training_points_top" value="/$(arg namespace)/$(arg app)/sim_mbes"/>     
      <param name="results_path" value="$(find uw_tests)/$(arg app)/$(arg dataset)/"/>     
      <param name="plot_gp_server" value="$(arg gp_plot_server)"/>     
      <param name="sample_gp_server" value="$(arg sample_sample_server)"/>     
    </node>
</launch>